modules/claims_api/app/sidekiq/claims_api/claim_uploader.rb

```diff
@@ -6,9 +6,7 @@
 require 'bd/bd'
 
 module ClaimsApi
-  class ClaimUploader
-    include Sidekiq::Job
-
+  class ClaimUploader < ClaimsApi::V2::DisabilityCompensationClaimServiceBase
```

Comment:
Seems weird to inherit from a specifically named service. The V2 nesting will make it feel even more weird when we move to V3, etc. I wonder if it would be worth it to extract the common methods into a base service outside of the V2 directory.